### PR TITLE
WV-2958 Additional Vegetation Indices

### PR DESCRIPTION
**Pull request by christof-wittreich**
## Description
This adds new customizations to the customizable HLS layers. Added are expressions for EVI, SAVI, MSAVI, NBR, NBR2, and TVI. Also added are a new separate layer for each of the above 6 expressions, 1 each for Landsat and Sentinel.

## How To Test
1. `git checkout wv-2958-more-veg-indices`
2. `npm ci`
3. `npm run watch`
4. Open a fresh instance of Worldview
5. Add one of the customizable HLS layers, either `HLS_Customizable_Landsat` or `HLS_Customizable_Sentinel`
6. Zoom in far enough to see the layer, and pick a date where imagery shows up onscreen
7. Open the layer options, click 'Customize', and select one of the new expression options
8. Verify that the layer imagery looks correct for the option selected
9. Repeat for all other newly added expressions
10. Repeat also for the other customizable HLS layer that wasn't previously picked, either Landsat or Sentinel
11. Check through each of the 12 non-customizable layers for the 6 new expressions, and verify they look and work correctly as well

### CODE REVIEW

**Review by PatchesMaps — APPROVED** [review]

File: config/default/common/config/metadata/layers/multi-mission/hls/HLS_SAVI_Sentinel.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+It is calculated using:
+
+`SAVI = ((NIR - R) / (NIR + R + L)) * (1 + L)`
+
+Specifically for Sentinel-2A and -2B:
+
+`SAVI = ((Band 8A – Band 4) / (Band 8A + Band 4 + 0.428)) * (1.428)`
+
+The image is applied with a divergent blue-green to brown color palette. It depicts areas with a lot of green leaf growth, indicating the presence of chlorophyll, in dark green colors.Chlorophyll reflects more infrared light and less visible light. Areas with some green leaf growth are in light greens, and areas with little to no vegetation growth are in shades of brown.
```
Comment (Tomcariello) [inline config/default/common/config/metadata/layers/multi-mission/hls/HLS_SAVI_Sentinel.md:13]: Missing a space here after the period. "...colors.Chlorophyll..."

File: config/default/common/config/metadata/layers/multi-mission/hls/HLS_SAVI_Landsat.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+**NOTE: This layer is undergoing beta testing.**
+
+The dynamically generated Soil Adjusted Vegetation Index (SAVI) imagery layer is used to correct Normalized Difference Vegetation Index (NDVI) for the influence of soil brightness in areas where vegetative cover is low. Surface Reflectance-derived SAVI is calculated as a ratio between the R and NIR values with a soil brightness correction factor (L) defined as 0.5 to accommodate most land cover types.
+
+It is calcuated using:
```
Comment (Tomcariello) [inline config/default/common/config/metadata/layers/multi-mission/hls/HLS_SAVI_Landsat.md:5]: "calcuated" is mispelled.

File: config/default/common/config/metadata/layers/multi-mission/hls/HLS_SAVI_Landsat.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`SAVI = ((NIR - R) / (NIR + R + L)) * (1 + L)`
+
+Specifically for Landsat 8 and 9:
+
+`SAVI = ((Band 5 – Band 4) / (Band 5 + Band 4 + 0.5)) * (1.5)`
+
+The image is applied with a divergent blue-green to brown color palette. It depicts areas with a lot of green leaf growth, indicating the presence of chlorophyll, in dark green colors.Chlorophyll reflects more infrared light and less visible light. Areas with some green leaf growth are in light greens, and areas with little to no vegetation growth are in shades of brown.
```
Comment (Tomcariello) [inline config/default/common/config/metadata/layers/multi-mission/hls/HLS_SAVI_Landsat.md:13]: "colors.Chlorophyll" missing a space after the period.

File: config/default/common/config/metadata/layers/multi-mission/hls/HLS_MSAVI_Sentinel.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+It is calculated using:
+
+`MSAVI = (2 * NIR + 1 – sqrt ((2 * NIR + 1)2 – 8 * (NIR - R))) / 2`
+
+Specifically for Sentinel-2A and -2B:
+
+`MSAVI = (2 * Band 8A + 1 – sqrt ((2 * Band 8A + 1)2 – 8 * (Band 8A – Band 4))) / 2`
+
+DThe image is applied with a divergent blue-green to brown color palette. It depicts areas with a lot of green leaf growth, indicating the presence of chlorophyll, in dark green colors. Chlorophyll reflects more infrared light and less visible light. Areas with some green leaf growth are in light greens, and areas with little to no vegetation growth are in shades of brown.
```
Comment (Tomcariello) [inline config/default/common/config/metadata/layers/multi-mission/hls/HLS_MSAVI_Sentinel.md:13]: "DThe" - remove the "D"

File: config/default/common/config/metadata/layers/multi-mission/hls/HLS_MSAVI_Landsat.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+It is calculated using:
+
+`MSAVI = (2 * NIR + 1 – sqrt ((2 * NIR + 1)2 – 8 * (NIR - R))) / 2`
+
+Specifically for Landsat 8 and 9:
+
+`MSAVI = (2 * Band 5 + 1 – sqrt ((2 * Band 5 + 1)2 – 8 * (Band 5 – Band 4))) / 2`
+
+The image is applied with a divergent blue-green to brown color palette. It depicts areas with a lot of green leaf growth, indicating the presence of chlorophyll, in dark green colors.Chlorophyll reflects more infrared light and less visible light. Areas with some green leaf growth are in light greens, and areas with little to no vegetation growth are in shades of brown.
```
Comment (Tomcariello) [inline config/default/common/config/metadata/layers/multi-mission/hls/HLS_MSAVI_Landsat.md:13]: "...dark green colors.Chlorophyll..." needs a space after the period.

**Review by Tomcariello — APPROVED** [review]

**Review by minniewong — APPROVED** [review]

**Pull request merged**